lms/urls.py

```diff
@@ -101,6 +101,7 @@
 
     url(r'^api/commerce/', include('commerce.api.urls', namespace='commerce_api')),
     url(r'^api/credit/', include('openedx.core.djangoapps.credit.urls', app_name="credit", namespace='credit')),
+    url(r'^rss_proxy/', include('rss_proxy.urls', namespace='rss_proxy')),
```

Comment:
Well, I didn't say it should be defaulted to 'off' -- just controllable :)  If the intention is for this to be always-on for all installations, then it's fine to ship as-is.